### PR TITLE
New dlgsfor graphics

### PR DESCRIPTION
**Pull request by maxwellfundi**

### CODE REVIEW

File: instat/dlgCombineforGraphics.vb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        End If
+        autoTranslate(Me)
+        TestOkEnabled()
+    End Sub
+
+    Private Sub InitialiseDialog()
+
+    End Sub
+
+    Private Sub Setdefaults()
```
Comment (dannyparsons) [inline instat/dlgCombineforGraphics.vb]: Capital D needed.

Comment (dannyparsons) [inline instat/dlgCombineforGraphics.vb]: Same in other dialogs.

**Comment by maxwellfundi** [comment]
Aha, okay

**Comment by maxwellfundi** [comment]
@dannyparsons  this is done

**Pull request merged**